TrackerConditions/src/TrackerStatusMaker.cc

```diff
@@ -59,11 +59,25 @@ namespace mu2e {
     for (auto const& row : tpls_p->rows()) estatus.insert(TrackerElementStatus(row.id(),tpls_p->sidMask(),row.status()));
     for (auto const& row : tpas_p->rows()) estatus.insert(TrackerElementStatus(row.id(),tpas_p->sidMask(),row.status()));
     for (auto const& row : tssl_p->rows()) estatus.insert(TrackerElementStatus(row.id(),tssl_p->sidMask(),row.status()));
-    for (auto const& row : tsss_p->rows()) estatus.insert(TrackerElementStatus(row.id(),tsss_p->sidMask(),row.status()));
+    unsigned ntotal = tpls_p->rows().size() + tpas_p->rows().size() + tssl_p->rows().size();
+    // tsss can have duplicate entries from tssl
+    for (auto const& row : tsss_p->rows()){
+      auto elem = TrackerElementStatus(row.id(),tsss_p->sidMask(),row.status());
+      auto iter = estatus.find(elem);
```

Comment:
I think this exposes a problem, that TrackerElementStatus operator < can fail when comparing 2 objects at different levels (say panel and straw).  ie SID 1,1,0 (straw) will match SID 1,1,0 (panel).   We want to allow overlapping Straw and Panel  status objects, so we want operator < to also look at the mask:
 bool operator < (const TrackerElementStatus& other) const { return mask_ < other.mask_ || (mask_ == other.mask_ && sid_ < other.sid_); }

Reply:
Probably true - if we merge this PR for now I can continue testing VST in the meantime and we can come back to fix this

Reply:
OK.  I'd like to fix this before we forget.  Lets merge this then I'll fix operator <.